portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,38 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> messages;
+  
+  @Override
+  public void init() {
+    messages = new ArrayList<String>();
+    messages.add("Hello, my name is Andre!");
+    messages.add("I would ask you your name, but I dont have user input yet!");
+    messages.add("This is a hardcoded message.");
+    messages.add("Eventually I will have real comments to show.");
+    messages.add("I'm running out of message ideas so I will stop.");
+  }
+   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String json = convertToJson(messages);
+    response.setContentType("application/jsom;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(ArrayList messages) {
```

Comment:
Doesn't this need to be `ArrayList<String>`? I didn't think this would compile as written.